pixi-multistyle-text.ts

```diff
@@ -148,6 +155,19 @@ export default class MultiStyleText extends PIXI.Text {
 		};
 	}
 
+	private getDropShadowPadding(): number {
+		let maxDistance = 0;
+		let maxBlur = 0;
+
+		for (let styleKey of Object.keys(this.textStyles)) {
```

Comment:
I'm not a huge fan of `for of`. Instead, I'd prefer `Object.keys(this.textStyles).forEach(...)`
Unless there's a huge gain of performance by using for of?